From ab1c1b1b1ee9d469d60a41eeede9b4b4cb06ea3d Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:59:37 +0800
Subject: [PATCH] Test case5 for Change Package section

Description: Add a new package in Package section (Add the "MdeModulePkg.dec") in TestPkg/FakeEmuSec/Sec.inf.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/FakeEmuSec/Sec.inf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/TestPkg/FakeEmuSec/Sec.inf b/TestPkg/FakeEmuSec/Sec.inf
index 0a10d74c0b..1ed1f40852 100644
--- a/TestPkg/FakeEmuSec/Sec.inf
+++ b/TestPkg/FakeEmuSec/Sec.inf
@@ -37,6 +37,7 @@
 [Packages]
   MdePkg/MdePkg.dec
   EmulatorPkg/EmulatorPkg.dec
+  MdeModulePkg/MdeModulePkg.dec
 
 [LibraryClasses]
   DebugLib
@@ -44,6 +45,7 @@
   PeiServicesLib
   PpiListLib
   BaseMemoryLib
+  HobLib
 
 [Ppis]
   gEfiTemporaryRamSupportPpiGuid
-- 
2.27.0.windows.1

